client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -705,11 +707,12 @@ public void testGetFieldMapping() throws IOException, InterruptedException {
         request.local(true); // <1>
         // end::get-field-mappings-request-local
 
-        {
+        final RequestOptions requestOptions = RequestOptions.DEFAULT.toBuilder().setWarningsHandler(WarningsHandler.PERMISSIVE).build();
```

Comment:
👍

Reply:
Followed your suggestion to add deprecated annotation to `GetFieldMappingsRequest#local(..)` methods and [removed](https://github.com/elastic/elasticsearch/pull/55014/files#diff-f588fdf24ad71ee0be8dd74dc447ee3aL173) it from the conversion.  It works better and this change to RequestOptions is no longer necessary.